custom/icds_reports/utils/aggregation_helpers/monolith/aww_incentive.py

```diff
@@ -84,7 +84,12 @@ def aggregation_query(self):
         /* update visits for cf cases (not in agg_ccs_record) */
         UPDATE "{tablename}" perf
         SET expected_visits = expected_visits + cf_data.expected,
-            valid_visits = valid_visits + cf_data.valid
+            valid_visits = valid_visits + cf_data.valid,
+            visit_denominator = round(expected_visits + cf_data.expected),
```

Comment:
visit_denominator is getting rounded in two places. Is there any concern that that would cause something like `2.4 + 2.4 = 4`

Reply:
It shouldn't. Visit denominator is recalculated from scratch here for all the rows that are affected. It uses the unrounded new value of expected visits, a decimal number composed of the unrounded old value plus the new additions.

Reply:
Ah. ok so that means that this `expected_visits` is referring to the newly set value, and `visit_denominator` is not being referenced from inside the `round`. That makes sense